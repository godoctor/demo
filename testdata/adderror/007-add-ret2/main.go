package main

import "fmt"

func bar() { //<<<<<adderror,9,1,9,1,pass
	if 3 < 5 {
		fmt.Println("Yes")
	} else {
		fmt.Println("No")
	}
}

func main() {
	bar()
	fmt.Println("Done")
}
