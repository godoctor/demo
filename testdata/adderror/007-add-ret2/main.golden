package main

import "fmt"

func bar() error { //<<<<<adderror,9,1,9,1,pass
	if 3 < 5 {
		fmt.Println("Yes")
		return nil
	} else {
		fmt.Println("No")
		return nil
	}
}

func main() {
	bar()
	fmt.Println("Done")
}
